shadowsocks-csharp/Controller/PolipoRunner.cs

```diff
@@ -20,6 +20,32 @@ class PolipoRunner
         static PolipoRunner()
         {
             temppath = Path.GetTempPath();
+            string[] commandLineArgs = Environment.GetCommandLineArgs();
+            for (int i = 0; i < commandLineArgs.Length; i++)
+            {
+                string commandLineArg = commandLineArgs[i];
+                if (commandLineArg.ToLower() == "/SameFolderForPolipo".ToLower())

```

Comment:
You should use String.Equals(string a, string b, StringComparison comparisonType) <https://msdn.microsoft.com/en-us/library/t4411bks(v=vs.110).aspx>